docs/usage/advanced-hooks.md

```diff
@@ -132,6 +132,10 @@ test('should increment counter after delay', async () => {
 })
 ```
 
+Wrapping `incrementAsync` in `act()` is not necessary since the state updates happen
+asynchronously during `await waitForNextUpdate()`. The async utils automatically wrap the waiting
```

Comment:
Can you please change "utils" to "utilities"? I try to avoid abbreviations in the docs to assist beginners and non-english speakers that might not be familiar with the terms (googling the real word will at least give them a definition).

Reply:
Not using abbreviations is a good idea. Fixed!